client/src/partials/services/services.html

```diff
@@ -117,7 +117,7 @@
               <div class="form-group" ng-class="{ 'has-error' : CreateForm.$submitted && CreateForm.cost_center_id.$invalid }">
                 <label class="control-label">{{ "FORM.LABELS.COST_CENTER" | translate }}</label>
```

Comment:
I've not looked through this entire PR as a review but it would be best for our updated files/ new code to use the latest `translate` standards. https://github.com/IMA-WorldHealth/bhima-2.X/issues/1074